### PR TITLE
Refactor gap css styles

### PR DESCRIPTION
**Pull request by asylves1**
# Description

Added default values that match tailwind gaps to `_variables.scss`.
replaced some of the values for `.25rem`, `.5rem` & `.75rem`

Resolves #(issue)

### CODE REVIEW

**Review by shawnyama — APPROVED** [review]

File: packages/client/hmi-client/src/assets/css/theme/_variables.scss [hunk 1]
```diff
@@ -1,3 +1,33 @@
...
+	--gap-16: 4rem;
+	--gap-20: 5rem;
+	--gap-24: 6rem;
+	--gap-28: 7rem;
+	--gap-32: 8rem;
+	--gap-36: 9rem;
+	--gap-40: 10rem;
+	--gap-80: 20rem;
+	--gap-84: 21rem;
+	--gap-88: 22rem;
```
Comment (YohannParis) [inline packages/client/hmi-client/src/assets/css/theme/_variables.scss:28]:
```suggestion
```
Those are not necessary.

File: packages/client/hmi-client/src/assets/css/theme/_variables.scss [hunk 2]
```diff
@@ -1,3 +1,33 @@
+:root{
+	// tailwind gaps as variables used in app
+	--gap-0-5: 0.125rem; 			// 2px
+	--gap-1: 0.25rem;					// 4px
+	--gap-1-5: 0.375rem;			// 6px
+	--gap-2: 0.5rem;					// 8px
+	--gap-2-5: 0.625rem;			// 10px
+	--gap-3: 0.75rem;					// 12px
+	--gap-3-5: 0.875rem;			// 14px
```
Comment (YohannParis) [inline packages/client/hmi-client/src/assets/css/theme/_variables.scss:9]: does `.5` not work compared top `-5`?

File: packages/client/hmi-client/src/assets/css/theme/designer/components/input/_autocomplete.scss [hunk 3]
```diff
@@ -69,7 +69,7 @@
     border-radius: 0;
     border-bottom-left-radius: 1.5rem;
     border-bottom-right-radius: 1.5rem;
-    padding-bottom: 0.25rem;
+    padding-bottom:  var(--gap-1);
```
Comment (YohannParis) [inline packages/client/hmi-client/src/assets/css/theme/designer/components/input/_autocomplete.scss:72]:
```suggestion
    padding-bottom: var(--gap-1);
```

File: packages/client/hmi-client/src/assets/css/theme/extensions/_autocomplete.scss [hunk 4]
```diff
@@ -21,7 +21,7 @@
 .p-float-label {
     .p-autocomplete-multiple-container {
         .p-autocomplete-token {
-            padding: .25rem 1rem;
+            padding:  var(--gap-1) 1rem;
```
Comment (YohannParis) [inline packages/client/hmi-client/src/assets/css/theme/extensions/_autocomplete.scss:24]:
```suggestion
            padding: var(--gap-1) 1rem;
```

**Review by YohannParis — COMMENTED** [review]
Just need to fix the extra space in front of all `var(--gap-1)`

File: packages/client/hmi-client/src/assets/css/theme/_variables.scss (hunk 2 above)
Comment (asylves1) [reply packages/client/hmi-client/src/assets/css/theme/_variables.scss:9]: Not for the variable name :(

**Review by YohannParis — CHANGES_REQUESTED** [review]
You still need to check **ALL** the `var(--gap-1)` because they **ALL** have a space in front of them.

**Review by YohannParis — APPROVED** [review]

**Pull request merged**